core/resources/scene/config.yaml

```diff
@@ -16,6 +16,12 @@ textures:
     water-texture:
         url: normals.jpg
         filtering: nearest
+
+# provide global fontfile names to be used for this stylesheet
+# these could be bundled with the tangram repo or could be loaded from system path
+# priority is given to bundled fonts
+fonts: [ Roboto-Regular.ttf, FiraSans-Medium.ttf]
```

Comment:
What about something like such: 
```yaml
fonts:
    font:
         name: Roboto
         face: Regular
    font: 
        name: FiraSans
        face: Medium
```
As we spoke last time, since we're able to know the names for the platform, we can have an a specific implementation for each platform to figure those name out.